### PR TITLE
Pooja Dev : Course List | Course Details

### PR DESCRIPTION
**Pull request by poojaapatil2117**
**Course List**

1. Adaptor 
       - extends RecyclerView.Adapter<CourceAdaptor.CourceViewHolder>
      
2. Modal
      Create Course Class where I Set getters and Setters.

3. Listener
      - creating one Interface in this package i.e ItemClickListner when we click on an item it will go to a detailed page of Cource

4. Cource List
     Create a List of Different Courses Using RecyclerView.
 
 

**Cource Details**

This page provides detailed information about the course.

- Name of Course.
- Course Curriculum
- Modes Of Training
- Instructor Info
- Payment Button and Cancel Button.

### CODE REVIEW

**Comment by PGW0808** [comment]
Approved

File: TutionApp/.idea/misc.xml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
```
Comment (ArcherInfotechInhouseTraining) [inline TutionApp/.idea/misc.xml:1]: add <?xml version="1.0" encoding="UTF-8"?>

Comment (ArcherInfotechInhouseTraining) [inline TutionApp/.idea/misc.xml:1]: add <?xml version="1.0" encoding="UTF-8"?> as first line

**Review by ArcherInfotechInhouseTraining — CHANGES_REQUESTED** [review]
make changes and pull request for merge

**Review by poojaapatil2117 — COMMENTED** [review]
Changes Done.

**Pull request closed without merging**